phpseclib/Net/SSH2.php

```diff
@@ -3290,12 +3300,15 @@ protected function reset_connection($reason)
      *
      * @see self::_send_binary_packet()
      * @param bool $skip_channel_filter
-     * @return string
+     * @return bool|string
      * @access private
      */
     private function get_binary_packet($skip_channel_filter = false)
     {
         if ($skip_channel_filter) {
+            if (!\is_resource($this->fsock)) {
+                throw new \InvalidArgumentException('fsock is not a resource.');
```

Comment:
Same thing as before - let's remove the `\`. Either all functions should have them or none should imho

Reply:
Removed. Here and everywhere else in the project